python_functionality/homework.py

```diff
@@ -11,45 +11,61 @@ def task_1_fix_names_start_letter(data: DT) -> DT:
     Make all `names` field in list of students to start from upper letter
 
     Examples:
-        fix_names_start_letters([{'name': 'Alex', 'age': 26}, {'name': 'denys', 'age': 89}])
+        fix_names_start_letters([{'name': 'Alex', 'age': 26},
+                                {'name': 'denys', 'age': 89}])
         >>> [{'name': 'Alex', 'age': 26}, {'name': 'Denys', 'age': 89}]
     """
-    pass
+    for dict_instance in data:
+        if dict_instance.get('name', None):
```

Comment:
you do not need to add the second parameter, 'None' is already the default

Reply:
Fixed.